client/web/src/site-admin/SiteAdminSlowRequestsPage.tsx

```diff
@@ -0,0 +1,281 @@
+import React, { ReactNode, useCallback, useEffect, useState } from 'react'
+
+import { mdiChevronDown, mdiContentCopy } from '@mdi/js'
+import classNames from 'classnames'
+import copy from 'copy-to-clipboard'
+import { RouteComponentProps } from 'react-router'
+import { map } from 'rxjs/operators'
+
+import { TelemetryProps } from '@sourcegraph/shared/src/telemetry/telemetryService'
+import {
+    Button,
+    Container,
+    Icon,
+    Link,
+    PageHeader,
+    Popover,
+    PopoverContent,
+    PopoverTrigger,
+    Position,
+    Text,
+    Tooltip,
+} from '@sourcegraph/wildcard'
+
+import {
+    FilteredConnection,
+    FilteredConnectionFilter,
+    FilteredConnectionQueryArguments,
+} from '../components/FilteredConnection'
+import { PageTitle } from '../components/PageTitle'
+import { Timestamp } from '../components/time/Timestamp'
+import { SlowRequestsResult, SlowRequestsVariables } from '../graphql-operations'
+
+import { SLOW_REQUESTS } from './backend'
+
+import { dataOrThrowErrors } from '@sourcegraph/http-client'
+
+import styles from './SiteAdminSlowRequestsPage.module.scss'
+import { requestGraphQL } from '../backend/graphql'
+
+export interface SiteAdminSlowRequestsPageProps extends RouteComponentProps, TelemetryProps {
+    now?: () => Date
+}
+
+export type SlowRequest = SlowRequestsResult['slowRequests']['nodes'][0]
```

Comment:
```suggestion
type SlowRequest = SlowRequestsResult['slowRequests']['nodes'][0]
```

I don't think you use this type outside of this file